app/scripts/lib/trust-signals/security-alerts-api.ts

```diff
@@ -40,16 +40,25 @@ export async function scanAddress(
 export async function scanAddressAndAddToCache(
   address: string,
   appStateController: AppStateController,
-  networkController: NetworkController,
+  networkController?: NetworkController,
+  chainId?: SupportedEVMChain,
 ): Promise<ScanAddressResponse> {
+  let chainID = chainId;
```

Comment:
This naming seems prone for future typos... the new variable should be named something different, so that it is clear.